src/com/dakl/Main.java

```diff
@@ -237,6 +245,7 @@ static CheckerBoard freshBoard() {
         return new CheckerBoard(board, red, black);
     }
 
+    // REVIEW : good usage of static methods
```

Comment:
This is a good usage of a static method (placed inside of the class itself) to create a pre-configured instance of the class (aka "factory methods")